src/wui/wui.c

```diff
@@ -119,7 +122,17 @@ static void wui_queue_cycle() {
 }
 static int process_wui_request() {
 
-    //if(wui.request == gcode)
-    marlin_json_gcode(wui.request);
+    if(strncmp(wui.request, "!cip ", 5) == 0){
+        uint32_t ip;
+        if(sscanf(wui.request + 5, "%u", &ip)){
+            eeprom_set_var(EEVAR_CONNECT_IP, variant8_ui32(ip));
```

Comment:
setting to eeprom and checking of the IP and other condition should be done only at one place for this and ini file

Reply:
Agreed. This will be implemented when ini file handling will be seperated from GUI.